src/Data/CallbackRequest/Pisa/pisaCallbackRequestDataClass.ts

```diff
@@ -1,22 +1,22 @@
 import { provideDataClass } from 'scrivito'
 import { pisaClient } from '../../pisaClient'
-import { DataIndexResponse, RawItem } from '../../types'
-
-export async function pisaCallbackRequestDataClass() {
-  const callbackRequestClient = await pisaClient('callback-request')
+import { DataConnection, DataIndexResponse, RawItem } from '../../types'
 
+export function pisaCallbackRequestDataClass() {
   // callback-request is more or less a "singleton". It only offers PUT, GET and DELETE.
   return provideDataClass('CallbackRequest', {
-    connection: {
-      index: async () =>
-        ({
-          results: [await callbackRequestClient.get('')],
-        }) as DataIndexResponse,
-      get: () => callbackRequestClient.get(''),
-      create: async (data) =>
-        callbackRequestClient.put('', { data }) as Promise<RawItem>,
-      update: async (_id, data) => callbackRequestClient.put('', { data }),
-      delete: (id) => callbackRequestClient.delete(id),
-    },
+    connection: pisaClient('callback-request').then(
+      (callbackRequestClient): DataConnection => ({
```

Comment:
* [ ] FYI Naming: if we name the parameter `apiClient` after its type (`ApiClient`), this code in particular becomes more readable. We'd have to use the same pattern everywhere, though. But I think this would be fine.